tests/stream/test_library.py

```diff
@@ -1,16 +1,10 @@
 from __future__ import absolute_import, unicode_literals
 
-import unittest
-
-import gobject
-gobject.threads_init()
```

Comment:
mopidy/audio/scan.py doesn't init threads, so this might crash without the threads init...